### PR TITLE
Add LevelUp

### PR DESCRIPTION
**Pull request by ilyakatz**

### CODE REVIEW

File: _data/finance.yml [hunk 1]
```diff
@@ -35,6 +35,12 @@ websites:
       img: kiva.png
       tfa: No
 
+    - name: LevelUp
+      url: https://www.thelevelup.com
+      twitter: levelupcom
```
Comment (Carlgo11) [inline _data/finance.yml]: That's not the right twitter handle. That account is for some Spanish gaming news site (http://www.levelup.com/).

**Comment by ilyakatz** [comment]
yes, you're correct. I must be seeing things, even the color scheme is drastically different :) fixed

**Comment by Carlgo11** [comment]
Eh we all make mistakes. Btw, I sent you a mail (I think). Can you confirm you got it?

**Comment by ilyakatz** [comment]
@Carlgo11 yups, I see it. Replying.

**Pull request merged**